tests/plugins/test_telemadrid.py

```diff
@@ -0,0 +1,11 @@
+from streamlink.plugins.telemadrid import Telemadrid
+from tests.plugins import PluginCanHandleUrl
+
+
+class TestPluginCanHandleUrlTelemadrid(PluginCanHandleUrl):
+    __plugin__ = Telemadrid
+
+    should_match = [
+        "https://www.telemadrid.es/",
+        "https://www.telemadrid.es/emision-en-directo/",
+    ]
```

Comment:
I forgot about VODs. Please add one VOD URL to the matcher tests, so that it's clear that this is supported by the plugin. For example this one, which isn't a particularly long URL string:
`https://www.telemadrid.es/programas/telenoticias-1/Telenoticias-1-02032023-2-2538066218--20230302042202.html`